native_client/deepspeech.cc

```diff
@@ -293,6 +342,7 @@ ModelState::infer(const float* aMfcc, unsigned int n_frames, vector<float>& logi
 {
   const size_t num_classes = alphabet->GetSize() + 1; // +1 for blank
 
+#ifndef USE_TFLITE
```

Comment:
It was easier and more readable to just have two big branches, each one being each implem